project/plugins.sbt

```diff
@@ -2,6 +2,12 @@
 //
 // e.g. addSbtPlugin("com.github.mpeltonen" % "sbt-idea" % "1.1.0")
 
+// Add EPFL Artifactory if on CI
+resolvers ++= {
```

Comment:
I think you're adding the same resolver 5 times, which may impact resolving speed.